src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -17,7 +17,7 @@ import ViewResults from './view-results/ViewResults';
 import headerImage from '../images/headerImage.png';
 
 const SkillsBuilderModal = () => {
-  const intl = useIntl();
+  const { formatMessage } = useIntl();
```

Comment:
I refactored the way the `formatMessage` is used in several components. I felt it was better to have it uniform throughout the app.